solutions/devsprint-1/DeliveryAppChallenge/Screens/RestaurantList/RestaurantListAPIDataSource.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  RestaurantListAPIDataSource.swift
+//  DeliveryAppChallenge
+//
+//  Created by Bruna Fernanda Drago on 10/11/21.
+//
+
+import Foundation
+
+typealias RestaurantListResult = (Result<[RestaurantListResponse], Error>) -> Void
+
+protocol RestaurantListAPIDataSourceProtocol {
```

Comment:
Somente um ajuste (sugestão)... Nesse caso, o protocolo ele não precisa especificar que ele é de uma Api, porque esse mesmo protocolo pode ser usado em um RestaurantListRealmDataSource, RestaurantListUserDefaultDataSource, etc. Então só deixar o nome mais genérico.
```suggestion
protocol RestaurantListDataSource {
```